exec/java-exec/src/main/java/org/apache/drill/exec/client/DrillClient.java

```diff
@@ -577,6 +581,8 @@ public Version getServerVersion() {
     checkArgument(type == QueryType.LOGICAL || type == QueryType.PHYSICAL || type == QueryType.SQL,
         String.format("Only query types %s, %s and %s are supported in this API",
             QueryType.LOGICAL, QueryType.PHYSICAL, QueryType.SQL));
+    //bingxing.wang.log
```

Comment:
Clean up needed.

Reply:
I am sorry to have brought you trouble.    I just want to merge drill 1.13.0 to my project 1.13.0 (what I have modified some code for personal use).  But it seems that I have made a mistake.

Reply:
Please ignore this commit. Very sorry to have brought you trouble.